test/runtests.jl

```diff
@@ -89,6 +89,15 @@ module wrapper11
 	
     @from "basic.jl" using A, C
 end
+module wrapper12
+	using FromFile
+	visible = [:foo, :quux, :A, :B, :C]
+	invisible = [:bar, :baz]
+
+	var"basic.jl" = 123
```

Comment:
So I think this test currently passes because `@from`'d files are imported into the root module (or `Main` when not in a package), whereas this is defined inside `Main.wrapper12`.

Reply:
I see. In that case I don't know how to write a test for this PR :) Feel free to take over, over close the PR and implement it however you see fit!

Reply:
I think it should be `Base.moduleroot(module12).eval(:(var"basic.jl" = 123))`.